WindowsServerDocs/networking/branchcache/BranchCache-Network-Shell-and-Windows-PowerShell-Commands.md

```diff
@@ -25,7 +25,7 @@ Windows PowerShell and netsh command references are at the following locations.
   
 -   [Netsh Commands for BranchCache in Windows Server 2008 R2](https://technet.microsoft.com/library/dd979561(v=ws.10))  
   
--   [BranchCache Cmdlets in Windows PowerShell](https://technet.microsoft.com/library/hh848392.aspx) for  Windows Server 2012 .  
+-   [BranchCache Cmdlets in Windows PowerShell](https://docs.microsoft.com/en-us/powershell/module/branchcache/?view=win10-ps)
```

Comment:
```suggestion
-   [BranchCache Cmdlets in Windows PowerShell](https://docs.microsoft.com/powershell/module/branchcache/?view=win10-ps)
```
It's better off removing the locale so when the localized version of this documentation is available, it will redirect you that version instead of the English one.